### ISSUE TITLE
Fix publisher colors

### ISSUE THREAD

**Issue by tfedor** [IsThereAnyDeal/IsThereAnyDeal-Issues#763]
Foreground:#131519, background:#424449. The contrast ratio is: 1,9:1. Text failed at Level AA. Text failed at Level AAA. Large text failed at Level AA. Large text failed at Level AAA.

All OK: foreground:#131519, background:#9EA2AD or foreground:#000000, background:#9297A1.

[Issue created by Bujno: 2017-06-14]
[Last updated on bitbucket: 2018-11-10]
